src/sc_crawler/vendors/_vultr.py

```
@@ -0,0 +1,600 @@
+        return StorageType.NVME_SSD
+    if api_storage == "block_storage":
+        return StorageType.NETWORK
+    disk_type = plan.get("disk_type")
+    if disk_type:
+        return _DISK_TYPES.get(disk_type)
+    return _DISK_TYPES.get(plan.get("type"))
+
+
+@cachier(separate_files=True)
+def _get_regions():
+    response = get("https://api.vultr.com/v2/regions", params={"per_page": 500})
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)